app/models/item_type.rb

```diff
@@ -1,4 +1,5 @@
 class ItemType < ActiveRecord::Base
+  belongs_to :creator, class_name: Service
```

Comment:
Thats true but what if someone wants to manually create a item type?
On Mar 16, 2016 7:48 PM, "David Faulkenberry" <notifications@github.com>
wrote:

> In app/models/item_type.rb
> <https://github.com/umts/aggressive-epsilon/pull/19#discussion_r56435698>:
>
> > @@ -1,4 +1,5 @@
> >  class ItemType < ActiveRecord::Base
> > +  belongs_to :creator, class_name: Service
>
> It'd be good to have a validation for the presence of creator. Then you
> can remove that guard on the hook method.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/umts/aggressive-epsilon/pull/19/files/6a2bab01e081a15e2c0e7ad5b9fd7240e6af586f#r56435698>
>